iati/utilities.py

```diff
@@ -154,6 +165,115 @@ def dict_raise_on_duplicates(ordered_pairs):
     return duplicate_free_dict
 
 
+def load_as_bytes(path):
+    """Load a file at the specified path into a bytes object.
+
+    Args:
+        path (str): The path to the file that is to be read in.
+
+    Returns:
+        bytes: The contents of the file at the specified location.
+
+    Raises:
+        FileNotFoundError (python3) / IOError (python2): When a file at the specified path does not exist.
+
+    Todo:
+        Ensure all reasonably possible `OSError`s are documented here and in functions that call this.
+
+        Add error handling for when the specified file does not exist.
+
+    """
+    with open(path, 'rb') as file_to_load:
+        data = file_to_load.read()
+
+    return data
+
+
+def load_as_dataset(path):
+    """Load a file at the specified path into a Dataset.
+
+    Args:
+        path (str): The path to the file that is to be read in.
+
+    Returns:
+        iati.Dataset: A Dataset object representing the contents of the file at the specified location.
+
+    Raises:
+        FileNotFoundError (python3) / IOError (python2): When a file at the specified path does not exist.
+
+        ValueError: When a file at the specified path does not contain valid XML.
+
+    Todo:
+        Ensure all reasonably possible OSErrors are documented here and in functions that call this.
+
+        Add error handling for when the specified file does not exist.
```

Comment:
As above.

Reply:
TODO should be removed since the error is a documented part of the API.

Reply:
05bc5b7c943e343907dfea600fe31d9e8b2d6abb